### ISSUE TITLE
Clone historic 2018 rates page for Pension - Survivors benefits

### ISSUE THREAD

**Issue by AnneHurley** [department-of-veterans-affairs/va.gov-team#10785]
URL: https://www.va.gov/pension/survivors-pension-rates/

Tasks:
- [x] Clone current Pension - Survivors rates page in Drupal. Selina
- [x] Copy over rates from legacy historic page, https://www.benefits.va.gov/PENSION/rates_survivor_pen17.asp
- [x] Copy edit and doublecheck rates and new math examples. Randi

Epic: #10544

Note - No 2019 rates page yet, should also create one.

## Acceptance Criteria
- [x] Page created in Drupal for 2018 historic rates for Pension and ready for SME approval

**Comment by lunascoop** [comment]
Hi @RLHecht this page is ready for review. http://preview-prod.vfs.va.gov/preview?nodeId=6859.

I also made update to the current rates page per our discussion in draft mode: preview-prod.vfs.va.gov/preview?nodeId=589

**Comment by RLHecht** [comment]
These numbers have been checked and updated. @lunascoop I am going to break out these tickets so we can close them once the work is ready for stakeholders and we'll have a new ticket for approval, changes and publishing.

**Comment by lunascoop** [comment]
@RLHecht the pages 2018, 2019 and current page have been updated to removed the year from the third bullet point. They are ready for review.

**Comment by RLHecht** [comment]
Thanks, @lunascoop Closing this ticket and continuing work in #12403 to send to SMEs and publish. FYSA @bethpottsVADEPO @MickinSahni
